devito/types/dense.py

```diff
@@ -167,7 +165,7 @@ def __staggered_setup__(self, **kwargs):
                     mask.append(-1)
                 else:
                     mask.append(0)
-            return tuple(mask)
+            return tuple(mask), True
```

Comment:
I thought this PR was droping the mask?

Reply:
Now that I think of it, I did but that's because I changed the FD accordingly too

Reply:
as you can see from the other changes, what this PR is dropping is the shape resizing based on the staggered mask

I also moved the  initialization
```
self.is_Staggered = ...
```
because attributes should be added to ``self`` only in the constructor, and never in subroutines